test/parachain/src/service.rs

```diff
@@ -191,21 +190,17 @@ pub fn run_collator(
 			block_announce_validator,
 		);
 
-		let (polkadot_future, polkadpt_task_manager) =
+		let (polkadot_future, polkadot_task_manager) =
 			polkadot_collator::start_collator(builder, id, key, polkadot_config)?;
 
-		// Make sure the polkadot task manager survives as long as the service.
-		let polkadot_future = polkadot_future.then(move |_| {
-			let _ = polkadpt_task_manager;
-			ready(())
-		});
-
 		task_manager
 			.spawn_essential_handle()
-			.spawn("polkadot", polkadot_future);
+			.spawn("polkadot-collator", polkadot_future);
```

Comment:
```suggestion
			.spawn("polkadot", polkadot_future);
```

This is the future of the Polkadot service.

Reply:
hmmm no I think it's the collator future: https://github.com/paritytech/polkadot/blob/master/collator/src/lib.rs#L263

It's a big loop that runs "collation work" when there are new notifications.

What I would call the "polkadot service" is the task manager that is passed as child right after this.

Reply:
I guess my vocabulary is still a big wrong :sweat: